osquery/examples/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+include(../../external/cmake/cmakelibs.cmake)
```

Comment:
It would be better to base the path from `${CMAKE_SOURCE_DIR}`, so if this CMakeLists.txt gets moved the path it's still valid.

Reply:
ok will do